### PR TITLE
Jenkins tutorial

### PR DESCRIPTION
**Pull request by erasche**
It was requested internally, hopefully it documents some of my knowledge for future people. @martenson anything you'd especially like me to mention?

### CODE REVIEW

File: topics/admin/tutorials/jenkins/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,490 @@
...
+These topics are outside the scope of a training or do not lend themselves well to interesting exercises, but some discussion and configuration information will be provided below documenting common issues and questions that can be useful if you start using Jenkins "in anger".
+
+## Updating Jenkins
+
+Jenkins updates regularly due to security issues. We recommend that you either:
+
+1. Regularly check the [Jenkins Changelog](https://jenkins.io/changelog/), and update the server whenever there are important security fixes
+2. OR include some sort of yum or apt auto-updater on whichever system that Jenkins is running on
+
+The downside of automatic updating is that Jenkins will restart and kill any jobs that had been running. Choose which works best for your environment. UseGalaxy.eu chose a completely automated setup because their jobs can be killed and few things are affected. UseGalaxy.org chose a more manual setup because their jobs are more critical, there if jobs fail, Galaxy pull request tests will fail unexpectedly and unhelpfully.
```
Comment (bgruening) [inline topics/admin/tutorials/jenkins/tutorial.md]: I think jenkins will only restart when no job is running, so it waits for running jobs, right? At least I have seen this happening.

Comment (bgruening) [reply topics/admin/tutorials/jenkins/tutorial.md]: Ok, was mentioned a few sentences below :)

**Comment by martenson** [comment]
Impressive! Nice @erasche

**Comment by erasche** [comment]
@martenson if you think of anything I missed, I'm happy to add in a follow up :)

**Pull request merged**